staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -214,6 +214,9 @@ type PodFailurePolicy struct {
 	// the backoffLimit. At most 20 elements are allowed.
 	// +listType=atomic
 	Rules []PodFailurePolicyRule `json:"rules" protobuf:"bytes,1,opt,name=rules"`
+	// Only allow pods to be started once the terminating pods have been
```

Comment:
// Create replacement pods only once the terminating pods reach the terminal phase.